docs/checkit.js

```diff
@@ -330,42 +338,43 @@ CheckIt.prototype.onAuthStateChanged = function(user) {
 };
 
 
-CheckIt.prototype.generateEmptyCalendar = function(calObj, $calendarDiv) {
+CheckIt.prototype.generateEmptyCalendar = function(calObj) {
     // Generate the html for an empty calendar of the calendar you want to 
     // display.
     var checkitApp = this;
     // Add the title of the calendar
-    $calendarDiv.append('<div id="calendarTitleHeading"><h1 class="page-header\
+    this.$calendarDiv.append('<div id="calendarTitleHeading"><h1 class="page-header\
                          text-center">'+calObj.state.title + '</h1></div>');
                         
     calObj.monthObjects.forEach (function(monthObj, index) {
         
         //the div ID is the monthID
-        $calendarDiv.append('<div class="monthframe" id=' + monthObj.monthId + '></div>');
+        checkitApp.$calendarDiv.append('<div class="monthframe" id=' + monthObj.monthId + '></div>');
         
         if (monthObj.monthIndex === 0 || index === 0) {
             var yearHeader = "<div class='page-header text-center' id='yearHeader'>" +
                 "<h2>"+monthObj.monthYear + "</h2>" +
                 "</div>";
-            $('#' + monthObj.monthId).append(yearHeader);
+            checkitApp.$calendarDiv.find('#' + monthObj.monthId).append(yearHeader);
         }
-        $('#' + monthObj.monthId).append(checkitApp.getTemplate());
+        checkitApp.$calendarDiv.find('#' + monthObj.monthId).append(checkitApp.getTemplate());
             
     });
     
 };
 
 CheckIt.prototype.fillCalendar = function(calObj) {
     // Fill an empty calendar with appropriate calendar data.
-    
+    var checkit = this;
+
     calObj.monthObjects.forEach (function(monthObj) {
         
-        var $monthId = $('#'+ monthObj.monthId);
+        var monthId = '#' + monthObj.monthId;
```

Comment:
You also could have changed this to be `var $monthId = checkit.$calendarDiv.find('#' + monthObj.monthId)`.